tests/test_model_derivatives.py

```diff
@@ -154,7 +154,7 @@ def f(value):
                 dphase = m.phase(toas, abs_phase=False) - phase
             except ValueError:
                 return np.nan * np.zeros_like(phase.frac)
-        return dphase.int + dphase.frac
+        return np.float64(dphase.int + dphase.frac)
```

Comment:
Hmmm.  That sum was long double before, right?  This type of conversion worries me a bit.  Because the int in this case can be tens of billions or maybe even hundreds of billions if people aren't careful.  Given 15 digits of decimal precision in a float64, that means that we only have 4-5 digits of precision for the fractional phase.  Is that accurate enough?

Reply:
Interestingly, I think I may have just run into an issue with TOA simulation that is exactly related to this.  I need to simulate good TOAs for a very fast MSP over 20+ years and here is what I got when I plotted the raw simulated TOAs.  Note the quantization.  I think that is likely the result of the a float64 representing phase being "saturated".  I will try to investigate (eventually).
![sim_prefit](https://github.com/nanograv/PINT/assets/132898/4aa08696-a684-4705-8364-1578b9f4b093)

Reply:
Yes, it was long double before, but this is just a test, not the actual code that is used.